backend/client-pkg/__tests__/services/integration/CanvasCreationServiceIntegrationTests.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Integration tests for the CanvasCreationService client package
```

Comment:
I am worried about these integration tests, specifically since we need them to pass for us to merge right? What if the server is down? Or is there a way to override that stuff?